### ISSUE TITLE
Router no longer honors domain list longer than 1.  Only first domain is routed.

### ISSUE THREAD

**Issue by florid-sojourner** [ash-project/ash_json_api#135]
**Describe the bug**
Multiple domains listed in the Ash router file only honors the first domain in the list.

**To Reproduce**
```
defmodule PhaseManagerWeb.RouterAshJsonApi do
  use AshJsonApi.Router,
    domains: [
        PhaseManager.Templates,
        PhaseManager.Projects
      ],
    json_schema: "/json_schema",
    open_api: "/open_api"
end
```

Although the Swagger is generated showing the resources of both domains, the second domain resources return:

```
{
  "errors": [
    {
      "code": "no_route_found",
      "id": "11c64745-d72b-417d-8bf9-d25499a182d8",
      "meta": {

      },
      "status": "404",
      "title": "NoRouteFound",
      "detail": "no route found"
    }
  ],
  "jsonapi": {
    "version": "1.0"
  }
}
```

Switch the order of the domains, and the other domain resources fail instead.

**Expected behavior**
Both domains' resources should have valid routes that respond to requests

** Runtime
 - Elixir version: 1.16.2
 - Erlang version: 26
 - OS: Ubuntu 22.04.4LTS
 - Ash version: 1.0.0-rc.5
 - any related extension versions

**Additional context**

**Comment by zachdaniel** [comment]
I believe this was just fixed: https://github.com/ash-project/ash_json_api/pull/134

**Comment by zachdaniel** [comment]
I will cut a release today or tomorrow. If you'd like you can try that commit and confirm.

**Comment by florid-sojourner** [comment]
Indeed, that is the same bug and that is the fix.    Thanks for your fast reply.

**Comment by zachdaniel** [comment]
Released in `v1.0.0-rc.6`
